javascripts/main.js

```diff
@@ -0,0 +1,33 @@
+let textInput = document.getElementById('text-input');
+
+window.addEventListener('load', function(){
+	CarLot.Inventory.loadInventory();
+});
+
+window.addEventListener('click', function() {
+	// console.log(event);
+	if (event.target.className === 'car col-sm-4') {
+		let targetCard = event.target;
+		CarLot.Inventory.clearSelectedClass();
+		targetCard.classList.toggle('selected');
```

Comment:
Seems that these two conditions (the 'if' and 'else if') have alot of repetitive code. Do you see a simple way you could dry this up?